sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -234,6 +239,8 @@ export class KeyClient {
     purgeDeletedKey(name: string, options?: PurgeDeletedKeyOptions): Promise<void>;
     releaseKey(name: string, target: string, options?: ReleaseKeyOptions): Promise<ReleaseKeyResult>;
     restoreKeyBackup(backup: Uint8Array, options?: RestoreKeyBackupOptions): Promise<KeyVaultKey>;
+    rotateKey(name: string, options?: RotateKeyOptions): Promise<KeyVaultKey>;
+    setKeyRotationPolicy(name: string, policy: KeyRotationPolicyProperties, options?: SetKeyRotationPolicyOptions): Promise<KeyRotationPolicy>;
```

Comment:
@heaths FYI in keyvault-certificates it's called `updateCertificatePolicy` - do you think we should do the same here? `update` felt weird because it's more of an upsert then an update I think? But happy to be consistent if you like `updateKeyRotationPolicy` better?

Reply:
Yeah, that does sound better and would be consistent.